pkgs/development/libraries/geoclue/default.nix

```diff
@@ -1,24 +1,29 @@
 { fetchurl, stdenv, intltool, pkgconfig, gtk-doc, docbook_xsl, docbook_xml_dtd_412, glib, json-glib, libsoup, libnotify, gdk_pixbuf
 , modemmanager, avahi, glib-networking, wrapGAppsHook, gobjectIntrospection
-, withDemoAgent ? false
+# doc build currently seems to be broken
+, enableDocs ? false
+, withDemoAgent ? true
```

Comment:
Demo agent should not be build by default, since the default use is for libraries. Override the value in the module when needed.

Reply:
I can do that, but then we'll build it twice. As is presumably libraries can just depend on `geoclue2.dev`, so I don' think this should affect closure size?

Reply:
`geoclue2.dev` is only used for build, the library is still in out. Maybe we could split `lib` to a separate output. Though building such a small program twice should not really matter.

Reply:
Okay, fine with me.